src/index.js

```diff
@@ -77,9 +77,22 @@ const css = (...styleDefinitions) => {
     return injectAndGetClassName(useImportant, styleDefinitions);
 };
 
+const injectedGlobals = new WeakSet();
```

Comment:
Unless I'm mistaken, this requires a polyfill. This project's probably not ready to drop Internet Explorer :)